### PR TITLE
Setup assembly plugin

### LINKED ISSUE #45: Make uberjar
So we are able to use it standalone. Such jar should be not used as a thirdparty dependency.

https://github.com/sbt/sbt-assembly

### PR DESCRIPTION
**Pull request by rexim**
### Description

Extract fatjar config from #203 because of #208 

Close #45

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,10 @@ And after that you can take a look at the Unit Test coverage
 results. Just open `target/scala-2.11/scoverage-report/index.html`
 with your favorite browser.
 
+### Build an uber jar ###
```
Comment (ForNeVeR) [inline README.md]: I propose letter `ü` here because it's cool.

File: README.md [hunk 2]
```diff
@@ -59,6 +59,10 @@ And after that you can take a look at the Unit Test coverage
...
+
+    $ sbt assembly
+
```
Comment (ForNeVeR) [inline README.md:67]: Also please specify a location of the überjar produced.

**Review by ForNeVeR — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -59,6 +59,14 @@ And after that you can take a look at the Unit Test coverage
 results. Just open `target/scala-2.11/scoverage-report/index.html`
 with your favorite browser.
 
+### Build an überjar ###
+
+To build an übérjàr run the following command
```
Comment (ForNeVeR) [inline README.md]: Ugh that's too hot!

**Review by ForNeVeR — CHANGES_REQUESTED** [review]

**Review by keddelzz — APPROVED** [review]

**Review by ForNeVeR — APPROVED** [review]

**Comment by rexim** [comment]
Thank you guys for the review!

**Pull request merged**